jck/build.xml

```diff
@@ -92,31 +92,42 @@
 			<available file="${JCK_ROOT_USED}" type="dir" />
 			<then>
 				<!--Obtain local and remote SHA to figure out if local materials are up-to-date--> 
-				<exec executable="git" dir="${JCK_ROOT_USED}" outputproperty="localSHA">
+				<exec executable="git" dir="${JCK_ROOT_USED}" failonerror="false" outputproperty="localSHA" resultproperty="return.code">
 					<arg line="rev-parse"/>
 					<arg line="HEAD"/>
 				</exec>
-				<exec executable="git" dir="${JCK_ROOT_USED}" outputproperty="remoteSHA">
```

Comment:
Having failonerror default (true) here is right, as if there is a connection to github issue we don't want the testing to continue.

Reply:
Added failonerror(true) as per comment.